rest/handler.go

```diff
@@ -482,31 +482,28 @@ func (h *handler) validateAndWriteHeaders(method handlerMethod, accessPermission
 	if ks != "" {
 		ksNotFound := base.HTTPErrorf(http.StatusNotFound, "keyspace %s not found", ks)
 		if dbContext.Scopes != nil {
-			// If scopes are defined on the database but not in th an empty scope to refer to the one SG is running with, rather than falling back to _default
+			// endpoint like /db/doc where this matches /db._default._default/
```

Comment:
I feel like it would be useful to add an addendum here like "(the check whether the default scope and collection are defined for the database is performed below)".

Reply:
Added comment.